test/simple.test.ts

```diff
@@ -43,6 +46,47 @@ integrationTestSkipOnWindows("forwarding signals to the child", async () => {
   expect(readAppmap()).toMatchSnapshot();
 });
 
+integrationTestSkipOnWindows(
+  "forwards SIGINT (ctrl-c) properly to the grandchild having active setInterval",
+  async () => {
+    // This test tests a fix (see: forwardSignals.ts) for a bug which occured
+    // when there is a script with active setInterval call and appmap-node
+    // runs this script indirectly with a node process: appmap-node node interval.js.
+    // We run the commands similarly as reported in:
+    // https://github.com/getappmap/appmap-node/issues/118
+    const child = spawn("npx", [getAppMapBinPath(), "node", "interval.js"], {
+      cwd: resolveTarget(),
+      shell: true,
+    });
+
+    await new Promise<void>(
+      (r) =>
+        child.stdout?.on("data", (chunk: Buffer) => chunk.toString().includes("heartbeat") && r()),
+    );
+
+    child.kill("SIGINT");
+
+    // child.kill does not kill all grandchildren, we have to
+    // send the kill signal to all of them. Otherwise Jest will
+    // not finish and we are not able to get the SIGINT in
+    // appmap-node process.
+    // https://krasimirtsonev.com/blog/article/Nodejs-managing-child-processes-starting-stopping-exec-spawn#killing-stopping-the-command
+    psTree(child.pid!, function (err, children) {
+      children.forEach((child) => process.kill(parseInt(child.PID), "SIGINT"));
+    });
+
+    await new Promise<void>((r) => {
+      const handler = (): void => {
+        child.removeListener("exit", handler);
```

Comment:
`removeListener` shouldn't be needed with `once` — you should be able to just do `child.once('exit', r)`.